inst/examples/demo/ui.R

```diff
@@ -12,6 +11,14 @@ fluidPage(
   title = paste0(share$title, " ", as.character(packageVersion("shinydisconnect"))),
   tags$head(
     includeCSS(file.path('www', 'style.css')),
+    HTML(
```

Comment:
In an effort to keep the code as shiny-y as possible, `tags$script()` over `HTML("<script>")` should be prefered